src/CwpSearchResult.php

```diff
@@ -65,8 +65,12 @@ public function __construct($terms = '', ArrayData $results = null)
             // Clean up the results.
             $matches = $results->Matches;
             foreach ($matches as $result) {
-                if (!$result->canView()) {
+                if ($result->hasMethod('isIndexable') && !$result->isIndexable()) {
```

Comment:
Hmmm, I was going to suggest a PHP class interface instead, but then you can't add this behaviour through DataExtension to existing DataObjects where you don't control the source, so ignore me :D

Reply:
okie dokie